<launch>
  <node pkg="rr_platform" type="motor_relay_node" name="motor_relay_node" output="screen" respawn="false">
    <param name="speed_topic" type="string" value="/speed" />
    <param name="steering_topic" type="string" value="/steering" />
    <param name="serial_port" type="string" value="/dev/ttyACM0" />
  </node>
</launch>
